### PR TITLE
Add minimal MultiPathDx prototype

### PR DESCRIPTION
**Pull request by cenacchi2000**
## Summary
- implement simple dataset loader for E-DAIC corpus
- add from-scratch logistic regression model
- create minimal orchestrator class and training script
- document setup and usage in README
- add .gitignore

## Testing
- `python train.py example_data example_data/labels/train_split.csv --epochs 20 --lr 0.1`

------
https://chatgpt.com/codex/tasks/task_e_684e9d3e5cfc83278cb4d7ead4eb7d0a

### CODE REVIEW

**Comment by cenacchi2000** [comment]
Initial setup
